open_schools_platform/organization_management/circles/filters.py

```diff
@@ -4,6 +4,15 @@
 from open_schools_platform.common.filters import BaseFilterSet, filter_by_ids
 from open_schools_platform.organization_management.circles.constants import CirclesConstants
 from open_schools_platform.organization_management.circles.models import Circle
+from open_schools_platform.student_management.students.selectors import get_students, get_student
+
+
+def circle_student_filter(queryset, name, value):
+    student = get_student(filters={"id": value}, empty_exception=True)
+    students = get_students(filters={"student_profile": student.student_profile})
+    students_circles = [student.circle.id for student in students]
+    queryset = queryset.filter(id__in=students_circles)
+    return queryset
```

Comment:
would be better to find directly by student_profile?

student__student_profile?